### ISSUE TITLE
BUG: Video texture alpha differs by platform

### ISSUE THREAD

**Issue by larsoner** [pyglet/pyglet#531]
This simple video code on Linux using [this file](https://github.com/LABSN/expyfun-data/blob/master/video/example-video.mp4?raw=true):

<details>

```
import numpy as np
import pyglet
from pyglet import gl
window = pyglet.window.Window()
player = pyglet.media.Player()
source = pyglet.media.StreamingSource()
media = pyglet.media.load('example-video.mp4')
player.queue(media)
player.play()


@window.event
def on_draw():
    window.clear()
    if player.source and player.source.video_format:
        tex = player.get_texture()
        global rawimage
        rawimage = tex.get_image_data()
        fmt = 'RGBA'
        assert rawimage.format == fmt
        pitch = rawimage.width * len(fmt)
        pixels = rawimage.get_data(fmt, pitch)
        data = np.frombuffer(pixels, np.uint8).reshape(rawimage.height, rawimage.width, len(fmt))
        print(data[..., -1])
        print(data.shape)
        gl.glEnable(gl.GL_BLEND)
        gl.glBlendFunc(gl.GL_SRC_ALPHA, gl.GL_ONE_MINUS_SRC_ALPHA)
        tex.blit(0, 0)
    else:
        pyglet.app.exit()


pyglet.app.run()
```

</details>

Produces a correct video output:

![Screenshot from 2022-01-07 11-17-17](https://user-images.githubusercontent.com/2365790/148573378-67ccb2f7-0d77-47a0-8c4d-7364f20edd5e.png)

And correct alpha channel in the texture if you pull the data (all 255's):
```
...
[[255 255 255 ... 255 255 255]
 [255 255 255 ... 255 255 255]
 [255 255 255 ... 255 255 255]
 ...
 [255 255 255 ... 255 255 255]
 [255 255 255 ... 255 255 255]
 [255 255 255 ... 255 255 255]]
...
```

But on Windows, it produces an all black image, and the alpha channel is all zeros:
```
[[0 0 0 ... 0 0 0]
 [0 0 0 ... 0 0 0]
 [0 0 0 ... 0 0 0]
 ...
 [0 0 0 ... 0 0 0]
 [0 0 0 ... 0 0 0]
 [0 0 0 ... 0 0 0]]
```
This was discovered because blitting does not work for this video/texture on Windows when blending is enabled (because alpha is zero).

This suggests that there is some difference in the FFMPEG->texture extraction on Windows compared to Linux.

**Comment by larsoner** [comment]
Turns out this is a WMF problem, not `ffmpeg`, so I re-titled this

**Comment by larsoner** [comment]
This was fixed by #534. No idea why [my magic tag in the PR description](https://github.com/pyglet/pyglet/pull/534#issue-1098418952) did not close this issue.
